### PR TITLE
Publisher: Fix publish logic in code

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

File: src/observability/consumer.go [hunk 1]
```diff
@@ -19,11 +19,20 @@ type SummaryConsumer struct {
 	Events        chan *types.SystemSummary
 }
 
-type SummaryStore struct {
+type Store struct {
```
Comment (rajaSahil) [inline src/observability/consumer.go]: nit: Can we define all global variables for observability in a struct and initialize it?

Comment (rajaSahil) [inline src/observability/consumer.go]: nit: Can we rename `Store` to `ConsumersStore`?

File: src/observability/consumer.go [hunk 2]
```diff
@@ -19,11 +19,20 @@ type SummaryConsumer struct {
...
 	Consumers map[*SummaryConsumer]struct{}
 	Mutex     sync.Mutex
 }
 
+var SummaryStore Store
```
Comment (rajaSahil) [inline src/observability/consumer.go]: nit: Can we rename `SummaryStore` to `SysSummary`?

File: src/observability/consumer.go [hunk 3]
```diff
@@ -38,22 +47,24 @@ func NewSummaryConsumer(req *ppb.SummaryRequest) *SummaryConsumer {
...
+func (sc *Store) RemoveConsumer(c *SummaryConsumer) {
 	sc.Mutex.Lock()
 	defer sc.Mutex.Unlock()
 
+	log.Info().Msgf("Consumer removed")
 	delete(sc.Consumers, c)
 }
 
-func (sc *SummaryStore) Publish(summary *types.SystemSummary) {
+func (sc *Store) Publish(summary *types.SystemSummary) {
```
Comment (rajaSahil) [inline src/observability/consumer.go]: Please add comments.

File: src/observability/consumer.go (hunk 2 above)
Comment (seswarrajan) [reply src/observability/consumer.go]: We can use as SysSummary

File: src/observability/consumer.go (hunk 1 above)
Comment (seswarrajan) [reply src/observability/consumer.go]: That will look good @rajaSahil 
We got some code clean up to do. We can take this up as part of that task.

**Review by rajaSahil — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**